Assets/MixedRealityToolkit.Tests/PlayModeTests/NearInteractionTouchableTests.cs

```diff
@@ -197,7 +203,7 @@ public IEnumerator NearInteractionTouchableVariant()
         [UnityTest]
         public IEnumerator NearInteractionTouchableVolumeVariant()
         {
-            var touchable = CreateTouchable<NearInteractionTouchableVolume>(objectScale);
+            var touchable = CreateTouchable<NearInteractionTouchableVolume>(Vector3.one);
```

Comment:
can we get rid of objectScale in the other unit test by using Vector3.one instead?

Reply:
Would rather keep the other tests the same, to not break them.

Reply:
The other test is using 1, 1, 0.1